### PR TITLE
Fix download for lynx

### PR DESCRIPTION
**Pull request by AndWac**
I am not sure whether the new COMPONENT_VERSION is correct.
I only downloaded and built lynx successfully with it.

BTW: Oracle userland gate offers the latest lynx (2.8.8) with two new patches.
OIs patch is not there anymore. As I don't use lynx I wasn't keen to update to
the new version, though.

### CODE REVIEW

**Comment by AndWac** [comment]
Hmm, I don't know what went wrong with the 2nd commit. I just merged the latest jpeg work into my branch...

**Comment by pyhalov** [comment]
Perhaps you should just go and update it to the latest minor version?

File: components/lynx/Makefile [hunk 1]
```diff
@@ -23,13 +23,13 @@
 include ../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		lynx
-COMPONENT_VERSION=	2.8.7
+COMPONENT_VERSION=	2.8.7rel.2
```
Comment (alarcher) [inline components/lynx/Makefile]: Should it not be numeric only ?

Comment (pyhalov) [inline components/lynx/Makefile]: If no, IPS_COMPONENT_VERSION should be also set

**Comment by AndWac** [comment]
Ok, I will try to update to 2.8.8 when time permits. Probably next week.

**Pull request closed without merging**